pkg/webhook/controlplane/ensurer_test.go

```diff
@@ -579,11 +516,26 @@ done
 				files = []extensionsv1alpha1.File{oldFile}
 			)
 
+			eContextK8s131 = gcontext.NewInternalGardenContext(
```

Comment:
Isn't possible to reuse the global `eContextK8s131` var?

Reply:
Then I need to change the scope of `ecrConfig, ecrBin` files because the files will contain these for the global `eContextK8s131 `variable.

Reply:
Sure, thanks!